.gitignore

```diff
@@ -22,3 +22,18 @@ scripts/glext.h
 
 # vscode cmake plugin
 build/*
+
+# vscode cmake plugin
+build/*
```

Comment:
This doesn't need to be in here twice. ^^

Reply:
oops